joomla/joomla-cms
# [#33826] Update server recommends and installs Joomla! 3.3 for php version < 5.3.10

### Tracker

http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=33826&start=0
### How to test
1. Use a PHP version < 5.3.10
2. Install vanilla Joomla! 2.5.20.
3. Change update server to STS for upgrade/update to Joomla! 3.x
4. Note the update notice to 3.3 including download package which references 3.3
5. apply patch
6. test again.
7. now we send a message that we can't upgrade.
8. reset the last_check_timestamp at #__update_sites to "0"
9. switch the PHP Version to (5.3.10 +)
10. try again.
11. Update should be show now.
### Note

It backports https://github.com/joomla/joomla-cms/pull/2802 and add a message if no update (URL) is available.
